libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -304,6 +316,39 @@ async def ainvoke(
         )
         return cast(ChatGeneration, llm_result.generations[0][0]).message
 
+    def _should_stream(
+        self,
+        *,
+        async_api: bool,
+        run_manager: Optional[
+            Union[CallbackManagerForLLMRun, AsyncCallbackManagerForLLMRun]
+        ] = None,
+        **kwargs: Any,
+    ) -> bool:
+        sync_not_implemented = type(self)._stream == BaseChatModel._stream
+        async_not_implemented = type(self)._astream == BaseChatModel._astream
+
+        # Check if streaming is implemented.
+        if (not async_api) and sync_not_implemented:
+            return False
+        if async_api and async_not_implemented and sync_not_implemented:
```

Comment:
Could you add a comment here to make sure we don't lose the `sync_not_implemented`?